active/teachingtip/TeachingTipSpec.md

```diff
@@ -533,15 +540,15 @@ unsealed runtimeclass TeachingTip : Windows.UI.Xaml.Controls.ContentControl
 | Title| * Semi-bolded. <br> * Text wraps at Close (X) Button and Container border. <br><br> <a href="url"><img src="Title.PNG" width="400"></a> |
 | Subtitle | * Text wraps at Close (X) Button and Container border. <br><br> <a href="url"><img src="Subtitle.PNG" width="400"></a> |
 | Content | * Can be customized to include text, images, videos, animation, checkboxes, hyperlinks, and any other XAML content. <br> * Will scroll if there is more content to show than tip height allows. See Scroll Bar. <br> * Placed below Subtitle and above Close/Action Button. <br><br> <a href="url"><img src="Content.PNG" width="400"></a> |
 | Close Button | * Will appear as an X Button in the top right corner by default and in the top left corner automatically for RTL languages. The close button may also be set to appear in the bottom right corner of the tip as a traditional button or be set to not show at all so that a custom close option may be implemented in the Content Area. <br> * If a tip is set to light-dismiss, no close button will appear at all. <br><br> <a href="url"><img src="HeaderClose.PNG" width="400"></a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <a href="url"><img src="FooterClose.PNG" width="400"></a> |
 | Action Button | * Allows the user to invoke a custom event. <br> * This is the only non-Close button provided out of the box. Other buttons may be implemented in the Content Area. <br><br> <a href="url"><img src="ActionButton.PNG" width="400"></a> |
 | Pointer | * Triangular extension of the tip body used to indicate that the tip is referring to on-screen UI element. <br> * Appears only when using a targeted tip. <br> * Prefers to center on target. <br> * Maintains a 12px margin from edges of the tip. <br> * Not animated. <br> * Will not have a shadow as shadows cannot yet be added to nonrectangular surfaces. <br><br> <a href="url"><img src="Pointer.PNG" width="400"></a> |
 | Icon | * Added to the left of the Title and Subtitle by default and automatically moved to the right for RTL languages. <br><br> <a href="url"><img src="Icon.PNG" width="400"></a> |
-| Bleeding Content | * Bleeding Content is media that stretches to the edges of a tip. <br> * Can be placed at the top or bottom of a tip. <br><br> <a href="url"><img src="BleedingContent.PNG" width="400"></a> |
+| Hero Content | * Hero Content is media that stretches to the edges of a tip. <br> * Can be placed at the top or bottom of a tip. <br><br> <a href="url"><img src="HeroContent.PNG" width="400"></a> |
```

Comment:
Quantifying a separate type of Content still feels odd to me.  Instead of this + `HeroPlacementProperty` can't it just re-use the existing notion's of `Header` and `Footer`?  Then someone could have both a banner on the top and some other accent on the bottom if they really wanted to easily.

Reply:
Thanks again for asking these kinds of questions, @michael-hawker. This content field was initially included to provide a solution for customers who place tutorial videos in the headers or footers of tips. This is often found in media editing or art related applications. Additionally, the appropriate use of image or gif hero content in tips was found to positively correlate with user engagement and satisfaction. The possibility of adding coexisting header, center, and footer content was considered but ultimately found to be overwhelming to users and overloaded the requests of customers. I believe these two primary content areas (body + header/footer) achieve customers requests without overloading the control for the sake symmetry or design "if's" considering other controls, such as a adaptive cards, may be used to achieve more custom visual scenarios. Do you or a customer you work with have an active need for both header and footer content in TeachingTip specifically?

Reply:
Sorry was at a conference last week.  I don't necessarily see an immediate need for both Header/Footer.  Just thinking that it's an easier concept to discover/use one or the other vs. having one specially named area and a separate enum controlling placement to understand when learning to use a new control.

Reply:
@michael-hawker, great feedback. I really appreciate the time you have invested into helping us make the best control possible. Your perspective is solid. 

I looked through our development history on this control (as some of it occured before our move to open source) and this was a path our design team also considered breifly. Ultimately, the ability to have both a header and footer simulatenously was scrapped because in application in was percieved as frequently distracting, visually overloaded, and created a sense of avoidable complexity for users without justifiable benefits or customers validated need. 

With that decision made, we are scoped to having a header **or** a footer which gives us two paths as far as API and implementation go:

1. Create one content area with a property for moving it.
2. Create two content areas and hope the developer uses only one. Which _could_  be the simplest in best case scenarioes but quickly turns messier than Option 1 otherwise: if both header and footer get set, we can use a magic rule to decide which is preferred, which leads to needing, an exposed API for the magic rule (which makes this essentially behave as Option 1 with avoidable complexity added). 

I am open to exploring another path forward if you seen one - it only needs enforce our designs data backed push for having only the header **or** footer area in use at any time.